users/admin.py

```diff
@@ -48,3 +71,51 @@ def is_online(self: "CustomUserAdmin", obj: User) -> bool | str:
 
     is_online.boolean = True
     is_online.short_description = "Online Status"
+
+    def get_urls(self: "CustomUserAdmin") -> list[path]:
+        urls = super().get_urls()
+        custom_urls = [
+            path(
+                "<int:user_id>/reactivate/",
+                self.admin_site.admin_view(self.reactivate_user_view),
+                name="reactivate_user",
+            )
+        ]
+        return custom_urls + urls
+
+    def reactivate_user_view(self: "CustomUserAdmin", request: HttpRequest, user_id: int) -> HttpResponseRedirect:
```

Comment:
This method also needs a check for valid time range to reactivate. Right now, there might be no link in admin to reactivate user, but it is easy to create /reactivate/ link by hand in browser, which is a security issue.